### PR TITLE
Adding troubleshooting section to README.md

### PR DESCRIPTION
**Pull request by asotona**
Hi,
based on an external contributor recent experience with javac benchmarking javac using jmh-jdk-microbenchmarks I've created a troubleshooting section of the README.md

Thanks,
Adam

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [ ] Change must be properly reviewed

### Reviewers
 * [Eric Caspole](https://openjdk.java.net/census#ecaspole) (@ericcaspole - Committer)

### Download
`$ git fetch https://git.openjdk.java.net/jmh-jdk-microbenchmarks pull/5/head:pull/5`
`$ git checkout pull/5`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +34,44 @@ For example:
...
+	--add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED \
+	--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED \
+	--add-exports=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED \
+	--add-exports=jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED \
+	-jar target/jmh-jdk-microbenchmarks-1.0-SNAPSHOT.jar [optional jmh parameters]
+
+### Execution of micros-javac benchmarks takes several hours
+
+micros-javac benchmarks consist of two sets of benchmarks: 
+ * `SingleJavacBenchmark` (which is parametrized) measures each single javac compilation stage in an isolated run. This benchmark is designed for exact automated performance regression testing and it takes several ours to execute completely.
```
Comment (mcimadamore) [inline README.md]:
```suggestion
 * `SingleJavacBenchmark` (which is parametrized) measures each single javac compilation stage in an isolated run. This benchmark is designed for exact automated performance regression testing and it takes several hours to execute completely. 
```

File: README.md [hunk 2]
```diff
@@ -34,3 +34,44 @@ For example:
...
+
+If you experience following exception during benchmarks execution:
+
+	java.lang.IllegalAccessError: superclass access check failed: class
+	org.openjdk.bench.langtools.javac.JavacBenchmark$2 (in unnamed module) cannot access class
+	com.sun.tools.javac.main.JavaCompiler
+
+It is caused by recently enabled Jigsaw enforcement and micros-javac benchmarks requirement to access several jdk.compiler module private packages. 
+
+Solution is to export required packages by adding following options to the command line:
```
Comment (mcimadamore) [inline README.md]: assuming this issue comes from reflective access, what about adding a single option to permit illegal access?

**Review by mcimadamore — COMMENTED** [review]
Looks a good troubleshooting guide. I've left some comments

File: README.md (hunk 2 above)
Comment (asotona) [reply README.md]: I thought it was also removed however it is only  deprecated, so to generally permit illegal access is definitely better solution. Thanks!

File: README.md (hunk 1 above)
Comment (asotona) [reply README.md]: fixed, thanks!

File: README.md (hunk 2 above)
Comment (asotona) [reply README.md]: fixed

**Review by ericcaspole — APPROVED** [review]
Looks good.

**Pull request merged**